### PR TITLE
Rename chimera

### PR DESCRIPTION
**Pull request by michaelybecker**
just a bunch of cosmetics -
1. rename chimera URLs to "launcher" etc
2. contact us link in every page

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -44,8 +44,9 @@ const config = {
       '/xr_design_and_accessibility',
     ],
     collapsedNav: ['/tutorials'],
-    links: [{ text: 'Pluto', link: 'https://pluto.app' }],
-    frontline: false,
+    collapsedNav: ['/platforms'],
```
Comment (thepelkus-too) [inline config.js]: Probably want these in a single array off of that key, yeah? (Looks like we went from collapsing one to collapsing the other.)

File: config.js [hunk 2]
```diff
@@ -26,14 +26,14 @@ const config = {
...
-    //   algoliaAdminKey: process.env.ALGOLIA_ADMIN_KEY,
-    // },
+    links: [{ text: 'CONTACT US', link: 'mailto:mait@plutovr.com' }],
+    search: {
+      enabled: true,
+      indexName: '',
+      algoliaAppId: process.env.GATSBY_ALGOLIA_APP_ID,
+      algoliaSearchKey: process.env.GATSBY_ALGOLIA_SEARCH_KEY,
+      algoliaAdminKey: process.env.ALGOLIA_ADMIN_KEY,
+    },
```
Comment (thepelkus-too) [inline config.js]: Is this doing anything right now?

File: config.js [hunk 3]
```diff
@@ -44,8 +44,9 @@ const config = {
       '/xr_design_and_accessibility',
     ],
     collapsedNav: ['/tutorials'],
-    links: [{ text: 'Pluto', link: 'https://pluto.app' }],
-    frontline: false,
+    collapsedNav: ['/platforms'],
+    links: [{ text: 'CONTACT US', link: 'mailto:mait@plutovr.com' }],
+    frontline: true,
```
Comment (thepelkus-too) [inline config.js]: What does this do? I tried to do a google and failed miserably. 😭

**Review by thepelkus-too — CHANGES_REQUESTED** [review]
Combine the collapse arrays, and I think we're good! I had a couple of other questions, but those are just for my own edification.

File: config.js (hunk 3 above)
Comment (thepelkus-too) [reply config.js]: `frontline` that is

File: config.js (hunk 1 above)
Comment (michaelybecker) [reply config.js]: oops! Absolutely, thanks.
EDIT: upon further reflection, I don't think anything should actually be collapsed as a default - removed.

File: config.js (hunk 2 above)
Comment (michaelybecker) [reply config.js]: yes! (or rather, will, very soon, Travis-gods willing)
image

File: config.js (hunk 3 above)
Comment (michaelybecker) [reply config.js]: from `README.MD` 😉: 
image

but, don't feel too bad - the option is misspelled _in the template_ (capitalized differently in `config.js`). Furthermore turning it on simply kills the submenus. Life is complicated.

File: config.js [hunk 4]
```diff
@@ -26,14 +26,14 @@ const config = {
...
-    //   enabled: true,
-    //   indexName: '',
-    //   algoliaAppId: process.env.GATSBY_ALGOLIA_APP_ID,
-    //   algoliaSearchKey: process.env.GATSBY_ALGOLIA_SEARCH_KEY,
-    //   algoliaAdminKey: process.env.ALGOLIA_ADMIN_KEY,
-    // },
+    links: [{ text: 'CONTACT US', link: 'mailto:mait@plutovr.com' }],
+    search: {
+      enabled: true,
+      indexName: 'dev_MAIT',
```
Comment (thepelkus-too) [inline config.js:32]: One more non-blocking question: is that index going to change in different environments? That's my reading of the `dev_` prefix, absent any other context.

**Review by thepelkus-too — APPROVED** [review]
LGTM!

**Pull request merged**